tfx/components/example_gen/import_example_gen/executor.py

```diff
@@ -19,6 +19,9 @@
 from absl import logging
 import apache_beam as beam
 import tensorflow as tf
+import pyarrow as pa
+import pyarrow.parquet as pq
+from apache_beam.io.filesystems import FileSystems
```

Comment:
let's use fileio https://github.com/tensorflow/tfx/blob/master/tfx/dsl/io/fileio.py

Reply:
don't think fileio has a match function? should I add it?